packages/admin/cms-admin/src/common/MasterMenu.tsx

```diff
@@ -25,6 +25,7 @@ type MasterMenuItemRoute = MasterMenuItemBase &
         type: "route";
         route?: RouteProps;
         to?: string;
+        badgeContent?: React.ReactNode;
```

Comment:
FYI: Barrel React imports will be removed in v8, you can remove it here

Reply:
https://github.com/vivid-planet/comet/pull/2431/commits/90effac002a863d97f3586a308688fe82b76380a